### DISCUSSION TITLE
Published ports on Synology not responding

### DISCUSSION THREAD

**Discussion by davidzakar** [felddy/foundryvtt-docker#156] [General]
I've got the 0.7.9 ("release") image working quite nicely in a container. But when I used the 0.8.0 tagged image, the container will start, but it doesn't actually return any data when I connect in as a user. Is this a problem with 0.8.0, or is there some environment setup that's different than 0.7.9 that I need to be aware of? I'll file a bug if this is unexpected, but didn't want to jump straight there without asking.

All of this is running on a Synology RS820+, if that's of any use.

**Comment by felddy** [comment]
Are you seeing anything of interest in the logs?  Does the container health status show `healthy`? 

You should see something like this with `docker ps`: 
```console
❱ docker ps 
CONTAINER ID   IMAGE                     COMMAND                  CREATED          STATUS                    PORTS                      NAMES
534a5b953b03   felddy/foundryvtt:0.8.0   "./entrypoint.sh res…"   43 seconds ago   Up 41 seconds (healthy)   0.0.0.0:30000->30000/tcp   foundryvtt-docker_foundry_1
```

**Comment by davidzakar** [comment]
First, thank you so much for the fast response.

OK, so I looked in the logs a little closer while trying to do the log export, and found this gem:
`FoundryVTT | 2021-03-11 15:59:39 | [[32minfo[39m] Server started and listening on port 30000`

I had my container configured for 30001, but now I'm wondering if I need to set the environment variable PORT for 30001, too? I didn't see that listed in README.md as an option, but maybe it's so standard it didn't need to be said? (Not a Docker guru, as you can see.)

ETA: Neither PORT nor FOUNDRY_PORT seemed to do anything for me.

**Reply by felddy** [reply]
The container will always listen on `30000`.  You then publish that port somewhere on the host: 
https://docs.docker.com/config/containers/container-networking/

If you had this working with `0.7.x` it should be the same with `0.8.0`.  If you are trying to run both simultaneously, then you'll need to publish the port to a different port.  e.g., `30001`.   There might be some magic happening on the Synology that auto-publishes exposed ports from the container.  If that is the case, you'll need to override that behavior.

**Comment by davidzakar** [comment]
That worked! Sorry I'm so clueless and wasted your time. Thank you again for this fantastic effort!

**Reply by felddy** [reply]
No problem.  Glad you got it working!
